lib/content/audio/train_is_boarding.ex

```diff
@@ -47,7 +47,14 @@ defmodule Content.Audio.TrainIsBoarding do
     end
 
     defp do_to_params(%{destination: destination, route_id: "Green-" <> _branch}, destination_var)
-         when destination in [:lechmere, :north_station, :government_center, :park_st, :kenmore] do
+         when destination in [
```

Comment:
Also realized I missed adding `:union_square` here. I believe the boarding messages for Union Sq would have been handled by the ad hoc logic anyway, but we have actual recordings for Union Square so we should use them. @losvedir @hannahpurcell